contracts/test/message-bus/SpyStateRootProvider.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity >=0.5.0 <0.6.0;
```

Comment:
we have `SpyAnchor` why do we need `SpyStateRootProvider`?

Reply:
Thank you.
 I have now removed *SpyStateRootProvider* contract.